backend/server/core/services/service.ts

```diff
@@ -0,0 +1,59 @@
+import { ActualTask } from '../interfaces/actualTask';
```

Comment:
I think it's better to create separate files for every function. Please name these files like `fuctionName.service.ts`